src/v/archival/ntp_archiver_service.cc

```diff
@@ -149,12 +173,101 @@ ss::future<> ntp_archiver::upload_loop() {
     }
 }
 
+ss::future<> ntp_archiver::sync_manifest_loop() {
+    ss::lowres_clock::duration backoff = _upload_loop_initial_backoff;
```

Comment:
Nit: Is there a reason for which the backoff for manifest sync should always be the same as the upload backoff?
If not, maybe we should define separate variables for the manifest sync backoff (_sync_manifest_loop_backoff and _sync_manifest_loop_max_backoff). This way the manifest sync would not be affected from tweaking the upload loop.

Reply:
got rid of backoff logic here